### PR TITLE
Add MUL operator to relay tflite frontend

### PR DESCRIPTION
**Pull request by apivovarov**
MUL operator is used in TFLite SSD Resnet 50 model (ssd_resnet_50_fpn_coco).
Model graph: [ssd_resnet_50_fpn_coco.tflite.pdf](https://www.dropbox.com/s/l2pirsrmq86gxau/ssd_resnet_50_fpn_coco.tflite.pdf?dl=0)
```
### ssd_resnet_50_fpn_coco summary:
# op_id: op_name - count
 0: ADD - 58
 2: CONCATENATION - 2
 3: CONV_2D - 110
14: LOGISTIC - 1
17: MAX_POOL_2D - 4
18: MUL - 42
22: RESHAPE - 14
32: CUSTOM - 1
34: PAD - 1
```

This PR transforms existing `convert_add` method to generic `_convert_elemwise` method (similar approach is used in tensorflow frontend code)
Existing `convert_add` and new `convert_mul` methods use `_convert_elemwise` method now.

I also created `_test_elemwise` method based on existing `_test_add` method.
Both `_test_add` and `_test_mul` use `_test_elemwise` method now

### CODE REVIEW

File: tests/python/frontend/tflite/test_forward.py [hunk 1]
```diff
@@ -24,7 +24,6 @@
 import numpy as np
 import tvm
 from tvm import relay
-from tvm.contrib import util
```
Comment (apivovarov) [inline tests/python/frontend/tflite/test_forward.py]: Import is not used. I deleted it

File: tests/python/frontend/tflite/test_forward.py [hunk 2]
```diff
@@ -144,8 +143,6 @@ def compare_tflite_with_tvm(in_data, in_name, input_tensors,
             for i in range(len(tflite_output)):
                 tvm.testing.assert_allclose(tflite_output[i], tvm_output[i], atol=1e-5, rtol=1e-5)
 
-        sess.close()
```
Comment (apivovarov) [inline tests/python/frontend/tflite/test_forward.py]: line 127 opens new `with tf.Session() as sess:` context block.
After python finishes `with tf.Session` block execution it calls`sess.__exit__()` method automatically which calls `self.close()` to close the session.
We do not need to close `sess` explicitly.

**Comment by apivovarov** [comment]
@srkreddy1238 @jroesch @FrozenGene Can you have a look?

**Comment by tqchen** [comment]
@kevinthesun please manage the PR

**Pull request merged**